chat/agent.js

```diff
@@ -4,6 +4,7 @@ try {
   var whiteSpace = /^\s+|\s+$|\s+(?=\s)/g;
   var aiMessages = {};
   var userScrolledUp = false;
+  var IS_STREAMING_ACTIVE = false; 
```

Comment:
this is not a big deal but can you use camel case?

Reply:
fix